ox-hugo.el

```diff
@@ -126,7 +126,11 @@ directory where all Hugo posts should go by default."
                    (:hugo-section "HUGO_SECTION" nil org-hugo-default-section-directory)
                    (:hugo-base-dir "HUGO_BASE_DIR" nil nil)
                    (:hugo-static-images "HUGO_STATIC_IMAGES" nil "images")
-                   (:hugo-code-fence "HUGO_CODE_FENCE" nil "t")))
+                   (:hugo-code-fence "HUGO_CODE_FENCE" nil "t")
+                   (:hugo-static-images "HUGO_STATIC_IMAGES" "images" nil)
```

Comment:
Looks like this got re-adding by mistake when you resolved the merge conflict. I had fixed the order of the default value. The correct order is:

> ;; KEY KEYWORD OPTION DEFAULT BEHAVIOR

So you have the `"images"`  and `nil` swapped.